c/makeotf/lib/cffread/CMakeLists.txt

```diff
@@ -1,4 +1,13 @@
 add_library(makeotf_cffread STATIC cffread.c)
 
-target_compile_definitions(makeotf_cffread PRIVATE $<$<CONFIG:Debug>:CFF_DEBUG=1> CFF_T13_SUPPORT=0)
+target_include_directories(makeotf_cffread PRIVATE AFTER $<$<COMPILE_LANGUAGE:CXX>:${ANTLR4_INCLUDE_DIRS}>)
+
+if(WIN32)
+    target_link_libraries(makeotf_cffread PUBLIC wsock32 ws2_32 libxml2)
```

Comment:
Given that they're repeated so often and could change with different compilers or C libraries it would probably better to use a variable to specify these three somewhere and refer to that.

Reply:
You're referring to `wsock32 ws2_32 libxml2`, right? 
I'll make a variable for these three, thanks 👍

Reply:
Yes, those dependencies